alpha/nodejs-chart/templates/NOTES.txt

```diff
@@ -0,0 +1,7 @@
+{{- if .Release.IsInstall }}
+Your Node.js app is building! To view the build logs, run:
+
+oc logs bc/{{ include "nodejs.name" . }} --follow
+
+Note that your Deployment will report "ErrImagePull" and "ImagePullBackOff" until the build is complete. Once the build is complete, your image will be automatically rolled out.
+{{- end }}
```

Comment:
The generated helm charts provided useful information on how to port forward along with the commands, I think we should probably provided that as well in addition to this text.

Reply:
The port forwarding isnt needed in openshift - the app is auto accesible. You need the port forwarding in vanilla kube only.

Reply:
We should touch based on that as for my setup I could not access until I started the port forwarding. (I am running on a separate machine)